spatialos-sdk/src/worker/locator.rs

```diff
@@ -43,51 +75,69 @@ impl Drop for Locator {
 }
 
 pub struct LocatorParameters {
-    pub project_name: CString,
+    pub project_name: Option<CString>,
```

Comment:
What does a `project_name` of `None` mean?

Reply:
It's unnecessary if you use the player authentication flow.